.github/workflows/deploy.yml

```diff
@@ -7,6 +7,9 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    permissions:
+    # this permission is mandatory for trusted publishing
+    id-token: write
```

Comment:
Either token, or trusted publishing don't mix.
Revert `.github/workflows/deploy.yml` here in favor of #1341.

Reply:
reverted as it has its own separate PR 👍🏼